devimint/src/gatewayd.rs

```diff
@@ -441,19 +441,38 @@ impl Gatewayd {
         Ok(txid)
     }
 
-    pub async fn close_all_channels(&self) -> Result<()> {
-        let channels = self.list_channels().await?;
-        for chan in channels {
-            let remote_pubkey = chan.remote_pubkey;
+    pub async fn close_channel(&self, remote_pubkey: PublicKey, force: bool) -> Result<()> {
+        let gateway_cli_version = crate::util::GatewayCli::version_or_default().await;
+        let mut close_channel = if force && gateway_cli_version >= *VERSION_0_9_0_ALPHA {
```

Comment:
Since we're backporting and including in `v0.8.1`, what do you think about updating here?